crates/oxc_linter/src/rules/eslint/no_duplicate_imports.rs

```diff
@@ -0,0 +1,315 @@
+use std::collections::HashMap;
+
+use oxc_ast::{
+    ast::{ExportNamedDeclaration, ImportDeclaration, ImportDeclarationSpecifier},
+    AstKind,
+};
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+use crate::{context::LintContext, rule::Rule};
+
+fn no_duplicate_imports_diagnostic(module_name: &str, span: Span) -> OxcDiagnostic {
+    OxcDiagnostic::warn(format!("'{}' import is duplicated", module_name))
+        .with_help("Merge the duplicated import into a single import statement")
+        .with_label(span)
+}
+
+fn no_duplicate_exports_diagnostic(module_name: &str, span: Span) -> OxcDiagnostic {
+    OxcDiagnostic::warn(format!("'{}' export is duplicated", module_name))
+        .with_help("Merge the duplicated exports into a single export statement")
+        .with_label(span)
+}
```

Comment:
it's probably worth useing `with_labels` and proving an array of spans so that we can highlight both of the import stmts

Reply:
I fixed it. Tell me if the wording are good

Reply:
looks good to me!